runtime/vm/UpcallVMHelpers.cpp

```diff
@@ -227,27 +227,41 @@ getInternalTypeFromSignature(J9JavaVM *vm, J9Class *typeClass, U_8 sigType)
  *
  * @param argType the predefined signature type
  * @param argValue the requested argument value
- * @return a U_32 value for the normalized argument
+ * @return a I_32 value for the normalized argument
  *
  * Note:
  * The argument normalization is only intended for the type less then 4 bytes
  * in java, which include boolean (1 byte), byte, char (2 bytes), short.
```

Comment:
This comment should be updated: "the type less then 4 bytes" -> "types no more than 4 bytes", etc.

Reply:
Fixed.

Reply:
The comment does not mention `int` or `float` which are handled by this function.

Reply:
We don't handle `int/float` in the function as mentioned at the default case:
```
getNormalizedArgValue(U_8 argType, I_32 argValue)
{
...
	default:
		/* Do nothing for the int/float type. */
		break;
	}
```

Reply:
You agreed with the "no more than 4 bytes" suggestion, so that includes `int` and `float`. The function does handle them (as it must, because `argType` may be `J9NtcInt` or `J9NtcFloat` at line 375).

You could shorten the comment by stopping at "no more than 4 bytes", but if you're going to list most of the Java types, you should name them all.

Reply:
> You could shorten the comment by stopping at "no more than 4 bytes", but if you're going to list most of the Java types, you should name them all.

I just restored the original comments as there is conversion required for `int/short`, which means we only handle `boo/byte/char/short` as mentioned in the description.